README.md

```diff
@@ -1 +1,126 @@
-# Leaderboard
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+> This project includes html, js and css file by webpack.
+> You are able to add your score here
+
+## 🛠 Built With <a name="built-with"></a>
+
+>js
+>html
+>css
+
+### Key Features <a name="key-features"></a>
+
+The ability to add your score the leaderbord
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+- Webhint
+- Styleint
+- Technologies used
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> In order to run this project you need:
+
+- Install code editor.
+- Install git.
+- Have github account.
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+- git clone https://github.com/LoviMadaan/Awesome-books.git
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Abdullah - AL-Khureif"></a>
+
+👤 **Abdullah AL-Khureif**
+
+- GitHub: [OnlyUnown](https://github.com/OnlyUnknown)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
OPTIONAL

- [ ] Kindly delete this template text from your readme this will make your readme more professional 👍🏿